### PR TITLE
Change in orientation

### PR DESCRIPTION
**Pull request by AetherPrior**
- Add .prettierrc file for client and server
- Change in orientation of Dashboard so it doesn't look like shit.
- Add cards to make it look neater than "not shit"

### CODE REVIEW

**Comment by AetherPrior** [comment]
image

**Comment by HarshvardhanJha1** [comment]
Very nice idea 👍

File: client/src/components/tabs/Dashboard.jsx [hunk 1]
```diff
@@ -1,112 +1,120 @@
...
-    overflow: "auto",
+    maxWidth: '95vw',
+    maxHeight: '26vh',
+    overflow: 'auto',
   },
-});
+  card: {
+    margin: '1vh',
+    width: '48%',
+    backgroundColor: '#DDDDDD',
```
Comment (arcane810) [inline client/src/components/tabs/Dashboard.jsx:41]: A lighter tone of gray would be better. Probably something as light as #F5F5F5

**Comment by arcane810** [comment]
image
Try adding some padding or space before heading here.
image
Make the Go button on the same line as the selections (or atleast add a <br/> before it)

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
+    "singleQuote": true
```
Comment (Aviral14) [inline .prettierrc:5]: It should be double quote? The complete project has double quote ig

Comment (AetherPrior) [reply .prettierrc:5]: Yeah will keep double quotes ig from now on

**Pull request merged**